ts/case-report.ts

```diff
@@ -51,9 +51,19 @@ const attributes: AttributeDefinition<Case>[] = [
   {
     title: "Case ID",
     addContents(object, fragment) {
-      fragment.appendChild(
-        makeNameDiv(object.id, undefined, urls.dimsum.case(object.id))
+      const container = document.createElement("div");
+      container.className = "flex flex-row space-x-1 items-center";
+      const caseNameDiv = makeNameDiv(
+        object.id,
+        undefined,
+        urls.dimsum.case(object.id)
       );
+      container.appendChild(caseNameDiv);
+
+      const misoUrl = urls.miso.assay(object.assayId);
+      addMisoIcon(container, misoUrl);
```

Comment:
The second parameter to `makeNameDiv` is a MISO URL, which takes care of all of this